lib/crypto/ed25519.go

```diff
@@ -0,0 +1,5 @@
+package crypto
+
+
```

Comment:
I would personally like to use a wrapper around libsodium to avoid any potential crypto bugs.